README.md

```diff
@@ -66,18 +66,22 @@ The `samlconf` script may take the following parameters:
            not provided, the default values will use DDF's parameters.
     
     OPTIONS
-           -i | --implementation path
+           -i path
                 The path to the directory containing the implementation's plugin and metadata.
                 The default value is `/implementations/ddf`.
                       
-           -d | --debug
+           -d
                 Boolean for whether or not to enable debug mode which enables more logging.
                 The default value is false.
 
-           -l | --lenient
-                When an error occurs, the SAML V2.0 Standard Specification requires an IdP to respond with a 200 HTTP status code and a valid SAML response containing an error <StatusCode>.
-                If the -l flag is given, this test kit will allow HTTP error status codes as a valid error response (i.e. 400's and 500's).
-                If it is not given, this test kit will only verify that a valid SAML error response is returned.
+           -l
+                When an error occurs, the SAML V2.0 Standard Specification requires an IdP to 
```

Comment:
❓ What is the purpose of making the lines shorter?

Reply:
It will wrap around instead of having people scroll horizontally to read everything